Library/Formula/osquery.rb

```diff
@@ -3,6 +3,7 @@
 class Osquery < Formula
   desc "SQL powered operating system instrumentation and analytics"
   homepage "https://osquery.io"
+  revision 1
```

Comment:
Should go below the `url` (It's always the last line of the initial block). If a local `brew audit osquery` doesn't flag it, don't worry about it, I can fix it on pull.